phpBB/styles/prosilver/theme/buttons.css

```diff
@@ -155,39 +155,5 @@
 /* Browser-specific tweaks */
 button::-moz-focus-inner {
 	padding: 0;
-	border: 0
-}
-
-/* Deprecated as of version 3.2
--------------------------------------------------*/
-.small-icon {
-	background-position: 0 50%;
-	background-repeat: no-repeat;
-	background-image: none;
-}
-
-.dropdown .small-icon {
-	background-position: 5px 50%;
-	padding: 5px;
-}
-
-.small-icon > a {
-	padding: 0 0 0 18px;
-}
-
-ul.linklist.bulletin > li.small-icon:before {
-	display: none;
-}
-
-.dropdown .small-icon > a {
-	display: block;
-}
-
-.rtl .small-icon {
-	background-position: 100% 50%;
-}
-
-.rtl .small-icon > a {
-	padding-left: 0;
-	padding-right: 19px;
+	border: 0;
```

Comment:
Not sure we can remove these. This requires a minor version change ie 3.4 which is not currently planned. You can make this change in master however

Reply:
i searched in the html files and couldn't find a call to it for 3.3.x so its not being used and was Deprecated as of version 3.2?
why cant it be removed in 3.3.x?

Reply:
It's a policy to deprecate 1 version then remove it next so If it was deprecated in 3.2 actually we should of removed it when we released 3.3 but I do not like doing it after a release as there could be some still using and we would be removing it breaking the code with out a big release. That's why I suggested to submit its removal on the master branch. It's not hurting anything be here

Reply:
Fix versions should have little to no changes to styling and html except for extreme bugs or unobtrusive visual fixes

Reply:
This is something I aim to change going forward with the new theme however by decoupling it's development from the main project and releasing as a style. But to do that we need to increase the front end dev team

Reply:
ok what about the ones that were commented out are they fine to be removed in this pr?